src/lib/accordion/Accordion.svelte

```diff
@@ -19,7 +19,7 @@
 
 <div
 	class={classes(
-		'border-gray-200 bg-white',
```

Comment:
i'd rather keep the background on this, a transparent accordion would be weird imo